src/types/provider/configuration.ts

```diff
@@ -20,3 +20,5 @@ export type RpcProviderOptions = {
   };
   batch?: false | number;
 };
+
+// export type ChainId = `0x${string}`;
```

Comment:
leftover comment?

Reply:
My bad, I was considering creating the type there before I changed the file. I will be removing that